mne_bids/dig.py

```diff
@@ -396,25 +387,38 @@ def _write_dig_bids(bids_path, raw, montage=None, acpc_aligned=False,
     if montage is None:
         montage = raw.get_montage()
     else:
-        # prevent transformation back to "head", only should be used
-        # in this specific circumstance
-        if bids_path.datatype == 'ieeg':
-            montage.remove_fiducials()
-        raw.set_montage(montage)
+        # transform back to native montage coordinates if given
+        with warnings.catch_warnings():
+            warnings.filterwarnings(action='ignore', category=RuntimeWarning,
+                                    message='.*nasion not found', module='mne')
+            raw.set_montage(montage)
```

Comment:
Have you considered setting the verbosity level of `set_montage()` to `'error'`? (I didn't take a look at this method, not sure if it even supports verbosity!)

Reply:
There could be other warnings though... wouldn't want to miss those